src/uwtools/tests/test_schemas.py

```diff
@@ -387,55 +447,106 @@ def test_schema_cdeps_streams(cdeps_config, section):
 # chgres-cube
 
 
-def test_schema_chgres_cube():
-    config = {
-        "execution": {"executable": "chgres_cube"},
-        "namelist": {"base_file": "/path", "validate": True},
-        "rundir": "/tmp",
-    }
+def test_schema_chgres_cube(chgres_cube_config):
     errors = schema_validator("chgres-cube", "properties", "chgres_cube")
     # Basic correctness:
-    assert not errors(config)
+    assert not errors(chgres_cube_config)
     # Some top-level keys are required:
     for key in ("execution", "namelist", "rundir"):
-        assert f"'{key}' is a required property" in errors(with_del(config, key))
+        assert f"'{key}' is a required property" in errors(with_del(chgres_cube_config, key))
     # Additional top-level keys are not allowed:
-    assert "Additional properties are not allowed" in errors({**config, "foo": "bar"})
+    assert "Additional properties are not allowed" in errors({**chgres_cube_config, "foo": "bar"})
+    # "rundir" must be a string:
```

Comment:
This string doesn't make the test case below. The test case below is checking to see if "rundir" is present and not a string. Should we add an additional test case? Something along the lines of 
`assert not "'rundir' must be a string" in errors(chgres_cube_config, {"rundir": 88})`

```suggestion
    # "rundir" is a required property:
```

Reply:
Nice catch, thanks! I'm fixing this manually and adding the type test.